Makefile

```diff
@@ -0,0 +1,22 @@
+lint:
+	npm run lint
+
+test:
+	npm run test
+
+end-to-end-test:
+	git clone https://github.com/weldr/welder-deployment
+	make -C welder-deployment/ weld-f25
+
+	sudo docker build -t weld/end-to-end:latest .
```

Comment:
This one will build welder-web, not end-to-end because the current folder is the welder-web.